app/helpers/tabs/observations_helper.rb

```diff
@@ -145,14 +145,20 @@ def observations_at_where_tabs(query)
     # these are from the observations form
     def define_location_tab(query)
       [:list_observations_location_define.l,
-       add_query_param(new_location_path(where: query.params[:user_where])),
+       add_query_param(new_location_path(where: where_param(query.params))),
        { class: tab_id(__method__.to_s) }]
     end
 
+    # Hack to use the :locations param if it's present and the :user_where
+    # param is missing.
+    def where_param(params)
+      params[:user_where] || params[:locations][0]
+    end
+
```

Comment:
It looks like Igor's example page (that has the `define_location_tab`) had the **incoming** `where` param you need:
`https://mushroomobserver.org/observations?where=Eberwhite+Nature+Area%2C+Washtenaw+Co.%2C+Michigan%2C+USA`

Does `params[:user_where] || params[:where]` not work? IIRC `user_where` would only come from an Advanced search.

Reply:
Oh, i notice you're using this below too.

Reply:
OH. I get it. There's a confusion here between `query.params` and ACTUAL params. Hang on, i'm going to commit what I believe is a fix.

Reply:
Fix seems to work.